repos/system_upgrade/common/actors/commonleappdracutmodules/files/dracut/85sys-upgrade-redhat/do-upgrade.sh

```diff
@@ -338,6 +346,14 @@ mount -o "remount,rw" "$NEWROOT"
 
 ##### do the upgrade #######
 (
+    # check if leapp previously failed in the initramfs, if it did return to the emergency shell
+    [ -f "$NEWROOT$LEAPP_FAILED_FLAG_FILE" ] && {
+        echo "Found file $NEWROOT$LEAPP_FAILED_FLAG_FILE"
+        echo "Leapp previously failed and cannot continue, returning back to emergency shell"
```

Comment:
error msg should be on `stderr` and should start with a prefix that clearly mark the msg as error one. see other error msgs in the module. so each `echo` should be e.g.:
```
echo >&2 "Error: ....."
```

Reply:
also add another echo msg to inform they should open the support support, as when this happens, we cannot provide any specific instructions what to do regarding the situation can be caused by whatever error.